content/ecosystem/data.toml

```diff
@@ -409,6 +413,7 @@ categories = ["networking"]
 name = "ezing"
```

Comment:
I'm 50/50 on this one - it's such a simple crate that I'm hesitant to remove it based on lack of updates. Is there a more widely used crate for easing functions these days?

Reply:
I don't really know, not my department 🤷🏻

Reply:
Had a quick look on Crates.io, there's a few that look a little more robust/active than this one. So probably fine to archive this.